contracts/proxy/transparent/ProxyAdmin.sol

```diff
@@ -29,32 +31,7 @@ contract ProxyAdmin is Ownable {
         ITransparentUpgradeableProxy proxy,
         address implementation,
         bytes memory data
-    ) public payable virtual {
-        // Owner checked by _upgrade
-        _upgrade(proxy, implementation);
-        Address.functionCallWithValue(address(proxy), data, msg.value);
-    }
-
-    /**
-     * @dev Upgrades `proxy` to `implementation`. See {TransparentUpgradeableProxy-_dispatchUpgradeTo}.
-     *
-     * Requirements:
-     *
-     * - This contract must be the admin of `proxy`.
-     */
-    function upgrade(ITransparentUpgradeableProxy proxy, address implementation) public virtual {
```

Comment:
Are we not keeping `upgradeTo` after all? I'm ok with that, but that was not what we agreed initially because of the plugin possibly using it. I guess that is what `PROXY_ADMIN_VERSION` is for.